en/docs/learn/configuring-inbound-provisioning.md

```diff
@@ -0,0 +1,53 @@
+# Configuring Inbound Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. 
```

Comment:
maintaining, and deleting

Reply:
Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065